test/integration/account_applications_test.rb

```diff
@@ -224,6 +224,72 @@ class AccountApplicationsTest < ActionDispatch::IntegrationTest
       end
     end
 
+    context "when the user already has some permissions" do
+      should "show the new and current permissions forms" do
+        # when I'm signed in as a user who can edit their own permissions
```

Comment:
I think that these comments would normally be context blocks. That way they show up in the stack trace if they fail. Do you think this can be rejigged a bit to make that work?

Reply:
I might be misunderstanding the suggestion, but breaking this one down into five context blocks (one per comment) would break this test? Throwing all five comments into the context might also make it a little long, even if condensed?

I felt the purpose here was a little different to what we'd typically use context blocks for in any case: it's breaking down a fairly long test by labelling what each chunk/action is doing, rather than saying what the overall context of the test is. I'm typically an anti-commenter, but it felt useful to add in some feature test style explanation here. Open to suggestions though!

Reply:
No, not suggesting a load of inline blocks, more that maybe the comments could be represented by the names of the existing ones. So instead of with comments
```
    context "when the user already has some permissions" do
      should "show the new and current permissions forms" do
```
Something like 
```
context "when a user with more than 8 permissions visits an app with more than 8 supported permissions"
should "show the new and current permissions forms"
```

Reply:
but that may be too wordy

Reply:
I've removed the comments but left the context as is, since the "with apps that have greater than eight permissions" is in an outer context. These match the style of the other tests now